i18n/en/design-system.json

```diff
@@ -14,10 +14,10 @@
   "global-footer-fandom-link-vertical-lifestyle": "Lifestyle",
   "global-footer-fandom-link-vertical-music": "Music",
   "global-footer-international-logo-home-of-fandom": "The Home of Fandom",
-  "global-footer-licensing-and-vertical-description": "__community__ is powered by __vertical__. Content is available under __license__.",
+  "global-footer-licensing-and-vertical-description": "__community__ is a Fandom __vertical__ Community. Content is available under __license__.",
   "global-footer-licensing-description": "Content is available under __license__.",
-  "global-footer-wikia-create-wiki-link-start-wikia": "Start a wikia",
-  "global-footer-wikia-create-wiki-description": "Find communities you love or create your own. Start Something epic.",
+  "global-footer-wikia-create-wiki-link-start-wikia": "START A WIKI",
```

Comment:
Yeah, already fixed. This was from just downloading the file from Crowdin, it was done like that earlier.